test/gomodgraph.test.ts

```diff
@@ -10,5 +10,6 @@ it('simple example: gomod parsing', async () => {
   const exampleGoMod = load(path.join('gomod', 'simple', 'gomodgraph'));
   const goModGraph = parseGoModGraph(exampleGoMod, 'github.com/spf13/jwalterweatherman');
   const expectedGraph = createFromJSON(JSON.parse(load(path.join('gomod', 'simple', 'expected-graph.json'))));
+  expect(goModGraph.toJSON()).toEqual(expectedGraph.toJSON())
```

Comment:
use dep-graph equals, comparing json is too brittle, graphs can order things differently but still be equal

Reply:
oh it's being done on line below, why this then?

Reply:
Was for testing locally, removed